logging/setup.py

```diff
@@ -51,9 +51,8 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core[grpc] >= 0.28.0, < 0.29dev',
-    'google-api-core >= 0.1.1, < 0.2.0dev',
-    'gapic-google-cloud-logging-v2 >= 0.91.0, < 0.92dev',
+    'google-cloud-core >= 0.28.0, < 0.29dev',
+    'google-api-core[grpc] >= 0.1.1, < 0.2.0dev',
```

Comment:
Unrelated, but what needs to happen to 1.0 this?

Reply:
What I'm doing right now- killing all gax usage to ensure that api-core covers all of its cases. Once it's done v1.0.0 is within reach.